src/components/CheckBox/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './CheckBox';
```

Comment:
[제안] 아래 같이 명명해서, 나중에 컴포넌트 찾을 때 수월하게 찾게 하는 건 어떠실까요? 네이밍을 바꿀 수 있다는 전제가 나중에는 유지보수 측면에서 파편화될까 우려되어 정규화하는 것이 좋을 것 같아서요!

```suggestion
export { default as Checkbox } from './CheckBox';
```